cernopendata_client/cli.py

```diff
@@ -80,7 +80,6 @@ def get_metadata(server, recid, doi, title, output_value):
     \b
     Examples:
       $ cernopendata-client get-metadata --recid 1
-      $ cernopendata-client get-metadata --recid 1 --output-value system_details.global_tag
```

Comment:
Here it is good to keep the line, since it is not demo'ed before... So just use something shorter, `--output-value title` for example.

Reply:
Done 👍

Reply:
... or `--output-value doi` if even more shortness is needed :wink: